toolset/setup/linux/frameworks/ulib.sh

```diff
@@ -28,6 +27,17 @@ if [ $? -ne 0 ]; then
    sudo apt-get install -y libcap2-bin
 fi
 
+# We need to install mongo-c-driver (we don't have a ubuntu package)
+#RETCODE=$(fw_exists ${IROOT}/mongo-c-driver.installed)
+#if [ "$RETCODE" != 0 ]; then
+  fw_get -O https://github.com/mongodb/mongo-c-driver/releases/download/1.1.10/mongo-c-driver-1.1.10.tar.gz
```

Comment:
So, this is causing issues. Apparently, github is responding with a head suggesting the filesize is [something] but the actual size of the file is [something else], which causes `curl` to return a status 18.